bin/combinetf2_fit.py

```diff
@@ -592,13 +599,15 @@ def main():
         fit_time = []
         for ifit in fits:
             ifitter.defaultassign()
+            ifitter.set_blinding_offsets(False)
```

Comment:
This might make sense to just fold into defaultassign()

Reply:
done